build-android-lib.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+function check_package() {
+    which "$1" 2>/dev/null || {
+      echo -e "Need to install $1."
+      exit 1
+    }
+}
+
+# Check required packages
+check_package gradle
+
+# Android SDK (Set your own path)
+[ -z "$ANDROID_HOME" ] && echo -e "Need to set ANDROID_HOME." && exit 1
+
+# GStreamer prebuilt libraries for Android
+# Download from https://gstreamer.freedesktop.org/data/pkg/android/
+[ -z "$GSTREAMER_ROOT_ANDROID" ] && echo -e "Need to set GSTREAMER_ROOT_ANDROID." && exit 1
+
+echo -e "Start to build NNStreamer library for Android."
+
+# NNStreamer root directory
+export NNSTREAMER_ROOT=$(pwd)
+echo -e "NNStreamer root directory: $NNSTREAMER_ROOT"
+
+cd $NNSTREAMER_ROOT/api/capi
+./modify_nnstreamer_h_for_nontizen.sh
+cd $NNSTREAMER_ROOT
+
+# Make directory to build NNStreamer library
+mkdir -p build_android_lib
+
+# Copy the files (native and java to build Android library) to build directory
+cp -r ./api/android/* ./build_android_lib
+
+# Get the prebuilt libraries and build-script
+pushd ./build_android_lib
+# TODO change to nnsuite repo later
+git clone https://github.com/jaeyun-jung/nnstreamer-build-ext.git
```

Comment:
Git clone might be too heavy for what you need. (You don't need the history)

There are options to skip history and extract the files you need only.

Reply:
changed to ```svn export ...```